pkg/image/prune/imagepruner.go

```diff
@@ -94,6 +96,9 @@ type PrunerOptions struct {
 	// KeepTagRevisions is the minimum number of tag revisions to preserve;
 	// revisions older than this value are candidates for pruning.
 	KeepTagRevisions int
+	// KeepOverSizeLimitImages indicates that images exceeding defined limits (openshift.io/Image)
+	// will not be considerate as candidates for pruning.
```

Comment:
In case of `keep-tag-revisions` and `keep-younger-then`, the *keep* says the truth (images below given age/revision will be kept). In case of over-sized images, the *keep* word won't safe them from being pruned (because the other flags have priority over this).